packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.tsx

```diff
@@ -45,7 +45,7 @@ export const RadioGroup = compose<RadioGroupType>({
         // @ts-ignore - TODO, fix typing error
         contextValue.buttonKeys = React.Children.map(children, (child: React.ReactChild) => {
```

Comment:
nit: should the name of this property change to reflect that it's not using buttonKeys anymore?

Also what happens if the value is the same across multiple radios? (though I guess the key didn't have any safeguards either. But a key is expected to be unique, at least)

Reply:
Updated to values to reflect V1 naming convention. 

We don't have safeguards to ensure that the values are unique, nor did old RadioGroup with the buttonKeys. We have a comment that says the value must be a unique key-identifier to prompt the user to pass in unique values. If there are multiple Radios with the same value, they would all show as selected when one is clicked.

Reply:
I think that's a bug. In theory we should be able to handle cases where values between multiple options are the same (for example a joke radio group having "yes, yes, yes" as options.